### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by EarlOfEgo**
Created some onboarding views and updated the dependencies a little bit. Also added some Java 8 language features(lambdas and method references)
Note: The assets are just placeholders.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -363,4 +363,12 @@
     <string name="fab_restore_viewonly">Restore view-only wallet</string>
     <string name="fab_restore_key">Restore wallet from private keys</string>
     <string name="fab_restore_seed">Restore wallet 25 word seed</string>
+    <string name="onboarding_security_title">Security</string>
+    <string name="onboarding_syncing_title">Syncing</string>
+    <string name="onboarding_seed_title">Seed Word</string>
+    <string name="onboarding_security_information">We take security seriously. Do not store your life savings int this app.</string>
+    <string name="onboarding_syncing_information">It takes forever bit it\'s worth it.</string>
```
Comment (jenniferberger) [inline app/src/main/res/values/strings.xml:370]: -bit +but

File: app/src/main/java/com/m2049r/xmrwallet/onboarding/OnBoardingActivity.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_on_boarding);
+
+        findViewById(R.id.buttonSkip).setOnClickListener(v -> finishOnboarding());
+        pager = findViewById(R.id.pager);
+        pagerAdapter = new OnBoardingAdapter(getApplicationContext());
+        pager.setAdapter(pagerAdapter);
+        int pixels = (int) TypedValue.applyDimension(
+                TypedValue.COMPLEX_UNIT_DIP,16, getResources().getDisplayMetrics());
```
Comment (jenniferberger) [inline app/src/main/java/com/m2049r/xmrwallet/onboarding/OnBoardingActivity.java:28]: -...DIP,16 + ...DIP,[space]16

File: app/src/main/res/drawable/dot_light.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <item>
+        <shape
+            android:innerRadius="0dp"
+            android:shape="ring"
+            android:thickness="8dp"
+            android:useLevel="false">
+            <solid android:color="#CDD1D9" />
+        </shape>
+    </item>
+</layer-list>
```
Comment (jenniferberger) [inline app/src/main/res/drawable/dot_light.xml:12]: style: missing newline

File: app/src/main/res/drawable/onboarding_dots.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:drawable="@drawable/dot_dark" android:state_selected="true" />
+    <item android:drawable="@drawable/dot_light" />
+</selector>
```
Comment (jenniferberger) [inline app/src/main/res/drawable/onboarding_dots.xml:6]: style: missing newline

File: app/src/main/res/layout/view_onboarding.xml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/moneroBlack"
+        android:textSize="24sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="@string/onboarding_security_title" />
+</android.support.constraint.ConstraintLayout>
```
Comment (jenniferberger) [inline app/src/main/res/layout/view_onboarding.xml:50]: style: missing newline

**Comment by m2049r** [comment]
superceded by & applied in #675 - thank you very much!

**Pull request closed without merging**